lua/damagelogs/shared/autoslay.lua

```diff
@@ -119,6 +132,16 @@ local function CreateCommand()
 end
 hook.Add("Initialize", "AutoSlay", CreateCommand)
 
+hook.Add( "TTTEndRound", "AutoSlay_CheckForMarked", function( res )
+		if not #Damagelog.markedForSlay > 0 then return end -- No need to do anything if no one is marked to be slain
+		for i = 1, #Damagelog.markedForSlay do
+			local data = Damagelog.markedForSlay[ i ]
+			Damagelog:AddSlays( data.admin, data.steamid, data.rounds, data.reason, data.target )
```

Comment:
Any reason to switch the command from `SwitchSlays` to `AddSlays`?

Reply:
What do you mean `SwitchSlays`?

Reply:
I meant SetSlays, don't know how it changed to SwitchSlays. Could it be possible that AddSlays is a modification in your own repository? I couldn't find anything in the original one.